packages/terra-slide-panel/tests/jest/SlidePanel.test.jsx

```diff
@@ -1,112 +0,0 @@
-import React from 'react';
```

Comment:
Do we want to remove these jest tests?

Reply:
We should keep the jest tests @amichaelparker